### PR TITLE
fixed user serializer which was causing login to fail

### PR DESCRIPTION
**Pull request by larryworm1127**

### CODE REVIEW

**Review by matticyang — COMMENTED** [review]
What is this for

File: lampsz/apis/serializers.py [hunk 1]
```diff
@@ -12,7 +12,7 @@ class Meta:
 class UserSerializer(serializers.ModelSerializer):
     class Meta:
         model = User
-        fields = ["id", "username", "email", "is_influencer"]
```
Comment (matticyang) [inline lampsz/apis/serializers.py:15]: should we create a public user serializer

File: lampsz/apis/serializers.py [hunk 2]
```diff
@@ -12,7 +12,7 @@ class Meta:
...
+        fields = ["id", "username", "password", "email", "is_influencer"]
```
Comment (matticyang) [inline lampsz/apis/serializers.py:15]: does this expose our password?

**Review by matticyang — COMMENTED** [review]
added 2 reviews, tks

**Review by matticyang — APPROVED** [review]
LGTM

File: lampsz/apis/serializers.py (hunk 2 above)
Comment (larryworm1127) [reply lampsz/apis/serializers.py:15]: it shouldn't if we only use it for serializing incoming data

**Pull request merged**